run.sh

```diff
@@ -42,6 +42,11 @@ elif [ $SYSTEMTESTS_SCENARIO = "APPSEC_CORRUPTED_RULES" ]; then
 elif [ $SYSTEMTESTS_SCENARIO = "APPSEC_CUSTOM_RULES" ]; then
     WEBLOG_ENV+="DD_APPSEC_RULES=/appsec_custom_rules.json"
 
+elif [ $SYSTEMTESTS_SCENARIO = "APPSEC_BLOCKING" ]; then
+    export RUNNER_ARGS="tests/appsec/waf/test_blocking.py"
```

Comment:
Can you remove this, and add @scenario("APPSEC_BLOCKING") in all classes in `tests/appsec/waf/test_blocking.py` ?